ios/MaasAccount/Sources/MaasAccount/Requirements/Components/ToggleView.swift

```diff
@@ -0,0 +1,86 @@
+import SwiftUI
+import Swappable
+
+public struct ToggleView<ToggledContent: View, UntoggledContent: View>: View  {
+
+    @Binding var isOn: Bool
+
+    private let toggledContent: ToggledContent
+    private let untoggledContent: UntoggledContent
+
+    public init(
+        isOn: Binding<Bool>,
+        @ViewBuilder toggledContent: () -> ToggledContent,
+        @ViewBuilder untoggledContent: () -> UntoggledContent
+    ) {
+        self._isOn = isOn
+        self.toggledContent = toggledContent()
+        self.untoggledContent = untoggledContent()
+    }
+
+    public var body: some View {
+        isOn
+            ? toggledContent.erased
+            : untoggledContent.erased
```

Comment:
maybe results in a nicer transition? 🤷‍♂️ 
```suggestion
        (isOn
            ? toggledContent.erased
            : untoggledContent.erased)
            .transition(.opacity)
```